src/components/Header/components/HeaderSearchButton.tsx

```diff
@@ -19,7 +24,8 @@ export interface HeaderSearchButtonProps {
 const HeaderSearchButton = chakra(
   ({ isMobile = false }: HeaderSearchButtonProps) => {
     const [isOpen, setIsOpen] = useState<boolean>(false);
-    const styles = useStyleConfig("HeaderSearchButton", { isOpen });
+    const isDarkMode = useColorModeValue(false, true);
```

Comment:
Use the `useColorMode` hook and get the `colorMode` string value. You'll have to make a boolean comparison for the "light" and "dark" values but it's better than using this function which is meant for css color values rather than boolean values.

Reply:
Ha.  I added `isDarkMode` becuase the intial styling was proving to be difficult.  However, in the end, I was able to do it all with the `_dark` object and I removed that prop from the style object.  So, I removed the code here as well.